localisation/00_wc_job_titles.csv

```diff
@@ -63,3 +41,316 @@ job_spiritual_void_group;Shadow Priest;Pr
+job_chancellor_arakkoa_group;Herald;Superviseur;Aufseher;;;;;;;;;;;x
+job_chancellor_female_arakkoa_group;Herald;Superviseur;Aufseherin;;;;;;;;;;;x
+job_marshal_arakkoa_group;Talon Guard;Garde Talon;Klauenw�chter;;;;;;;;;;;x
+job_marshal_female_arakkoa_group;Talon Guard;Garde Talon;Klauenw�chterin;;;;;;;;;;;x
+job_treasurer_arakkoa_group;Scrollkeeper;Parcheminier;H�ter der Pergamente;;;;;;;;;;;x
+job_treasurer_female_arakkoa_group;Scrollkeeper;Parcheminier;H�terin der Pergamente;;;;;;;;;;;x
+job_spymaster_arakkoa_group;Watcher;Observateur;Beobachter;;;;;;;;;;;x
+job_spymaster_female_arakkoa_group;Watcher;Observateur;Beobachterin;;;;;;;;;;;x
+job_chancellor_centaur_group;Spokesman;;Geschichtsschreiber;;;;;;;;;;;x
+job_chancellor_female_centaur_group;Spokeswoman;;Geschichtsschreiberin;;;;;;;;;;;x
+job_marshal_centaur_group;Warlord;Seigneur de Guerre;Kriegsf�rst;;;;;;;;;;;x
+job_marshal_female_centaur_group;Warlord;Seigneuse de Guerre;Kriegsf�rstin;;;;;;;;;;;x
+job_treasurer_centaur_group;Drudge;Drudge;Sklavenmeister;;;;;;;;;;;x
+job_treasurer_female_centaur_group;Drudge;Drudge;Sklavenmeisterin;;;;;;;;;;;x
+job_spymaster_centaur_group;Torturer;Tortionnaire;Peiniger;;;;;;;;;;;x
+job_spymaster_female_centaur_group;Torturer;Tortionnaise;Peinigerin;;;;;;;;;;;x
+job_chancellor_demonic_group;Subduer;Soumissionnaire;Unterwerfer;;;;;;;;;;;x
+job_chancellor_female_demonic_group;Subduer;Soumissionnaire;Unterwerferin;;;;;;;;;;;x
+job_marshal_demonic_group;General;G�n�ral;General;;;;;;;;;;;x
+job_marshal_female_demonic_group;General;G�n�rale;Gener�lin;;;;;;;;;;;x
+job_treasurer_demonic_group;Overseer;Superviseur;Aufseher;;;;;;;;;;;x
+job_treasurer_female_demonic_group;Overseer;Superviseur;Aufseherin;;;;;;;;;;;x
+job_spymaster_demonic_group;Painmaster;;Inquisitor;;;;;;;;;;;x
+job_spymaster_female_demonic_group;Painmistress;;Inquisitorin;;;;;;;;;;;x
+job_chancellor_draenei_group;Proconsul;;Prokonsul;;;;;;;;;;;x
+job_chancellor_female_draenei_group;Proconsul;;Prokonsulin;;;;;;;;;;;x
+job_marshal_draenei_group;High Vindicator;;Hoher Verteidiger;;;;;;;;;;;x
+job_marshal_female_draenei_group;High Vindicator;;Hohe Verteidigerin;;;;;;;;;;;x
+job_treasurer_draenei_group;Chief Artificer;;Erster Fachmann;;;;;;;;;;;x
+job_treasurer_female_draenei_group;Chief Artificer;;Erste Fachfrau;;;;;;;;;;;x
+job_spymaster_draenei_group;Rangari Prime;;Erster Rangari;;;;;;;;;;;x
+job_spymaster_female_draenei_group;Rangari Prime;;Erste Rangari;;;;;;;;;;;x
+job_chancellor_dragon_group;Broodlord;;Brutmeister;;;;;;;;;;;x
+job_chancellor_female_dragon_group;Broodlady;;Brutmeisterin;;;;;;;;;;;x
+job_marshal_dragon_group;General;;Kommandeur;;;;;;;;;;;x
+job_marshal_female_dragon_group;General;;Kommandeurin;;;;;;;;;;;x
+job_treasurer_dragon_group;Majordomo;;Majordomus;;;;;;;;;;;x
+job_treasurer_female_dragon_group;Majordomo;;Majordomus;;;;;;;;;;;x
+job_spymaster_dragon_group;Emissary;;Gesandter;;;;;;;;;;;x
+job_spymaster_female_dragon_group;Emissary;;Gesandte;;;;;;;;;;;x
```

Comment:
Emissary title is used by Chancellors. I suggest Saboteur.
https://wow.gamepedia.com/Azure_Saboteur